src/vos/tests/vos_cmd.c

```diff
@@ -807,7 +807,7 @@ split_cmd_args(const char *arg0, const char *cmd)
 	char  *dest;
 	char  *src;
 	char   last;
-	char  *saveptr;
+	char  *saveptr = NULL;
```

Comment:
This fixes an obscure bug with some versions of gcc where gcc thinks `strtok_r` has not initialized the pointer. Introduced by the additional linker flags in `compiler_setup.py`